BrickKit/app/build.gradle

```diff
@@ -33,6 +35,7 @@ android {
 }
 
 dependencies {
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.72'
```

Comment:
you don't need this w/ Kotlin 1.4... also shouldn't be using 1.3.72 any more..

Reply:
done